rpm/rpm_test.go

```diff
@@ -146,13 +156,6 @@ func TestWithRPMTags(t *testing.T) {
 	assert.Equal(t, info.Description, description)
 }
 
-func TestRPMVersionWithDash(t *testing.T) {
```

Comment:
this test used to check this because rpmbuild failed if version had a `-` on it...